NetSuiteProject/Exercise.js

```diff
@@ -2,22 +2,65 @@
 function add(x,y){
 	return x + y;
 };
+
+function addition(x,y){
+	if(typeof add (x,y) === "string"){
+	alert("This function only accepts Integers")
+	}
+	else {return x + y;}
+}
+
+
 //subtract
 function subtract(x, y){
 	return (x - y);
 };
+
+function subtraction(x, y){
+	if(isNaN(subtract(x,y))  === true){
```

Comment:
Because when I used typeof on subtraction I kept on getting "undefined". Therefore I had to find another operator that allowed me to satisfy the criteria. In subtraction I used isNaN to satisfy my if then statements.